src/summersplash2022/Content.tsx

```diff
@@ -0,0 +1,362 @@
+                    en unaturlig situasjon, og man blir ikke godt kjent med
+                    hverandre. Etter at vi har vurdert alle søknadene inviterer
+                    vi utvalgte kandidater til en kaffeprat. Dette er en
+                    uformell samtale hvor målet er å bli bedre kjent med
+                    hverandre, med fokus på både det faglige og det personlige.
+                    Og nei – du er selvsagt ikke nødt til å drikke kaffe.
+                  </p>
+                  <p className={style.section5ParagraphLeft}>
+                    Dersom du får jobbtilbud og takker ja, inkluderes du straks
+                    i Variant. Du får tilgang til vår Slack, og mulighet til å
+                    delta på faglige og sosiale arrangementer. Dette inkluderer
+                    blant annet spill- og fagkvelder, nyttårskalas og
+                    variantdager, som er fine muligheter til å bli bedre kjent
+                    før sommerjobben starter i juni.
+                  </p>
+                  <div className={style.Even}>
+                    <img src={Even} className={style.EvenPicture} alt="Even" />
+                  </div>
+                </div>
+                <div className={style.section5Div2}>
+                  <div className={style.timeLine}>
+                    <TimeLine />
+                  </div>
+
+                  <div className={style.søknadstips}>
+                    <h4 className={style.section5HeadingH4}>Søknadstips:</h4>
+                    <p className={style.section5Paragraph}>
+                      Vi setter pris på en søknad med CV, søknadsbrev og
+                      karakterutskrift. Det viktigste for oss er å få et
+                      helhetlig bilde. Både av deg som person, din eksisterende
+                      kompetanse og dine ambisjoner. Så hvem er du og hvorfor
+                      søker du sommerjobb i Variant? Vi trenger mennesker som
+                      bryr seg om å skape en bedre hverdag. Er det deg?
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </section>
+          </div>
+
+          <section className={style.section6} id="soksommerjobb">
+            <h3 className={style.section6Heading}>Søk sommerjobb</h3>
+            <div className={style.section6Flex}>
+              <div className={style.section6Text}>
+                <p className={style.section6Paragraph}>
+                  Vi håper du søker, og vi ser frem til å bli bedre kjent med
+                  deg. Har du spørsmål om sommerjobben eller Variant? Ta gjerne
+                  kontakt med meg.
+                </p>
+                <p className={style.section6Paragraph2}>
+                  - Marius Krakeli, Utvikler og rekrutteringsansvarlig
+                </p>
+                <div className={style.contactInfo}>
+                  <div>
+                    <p className={style.section6Underline}>41637572</p>
```

Comment:
Her tenker jeg det er greit å lenke til telefon direkte samme som med epost.

```suggestion
                    <a href="tel:41637572">41637572</a>
```